### DISCUSSION TITLE
Valorant, CS:GO & Original Wallpapers (Redesigned)

### DISCUSSION THREAD

**Discussion by jacksyrm** [Atlas-OS/Atlas#10] [Community Artwork]
# What do they look like?
Here are some of the images of the wallpapers. I have made 4 variants of each colour, and of each game. I have also decided to redesign the original wallpaper theme which was created the first time, to give it a more modern/clean feel to it.
# Previews
[CS:GO Previews](https://i.imgur.com/1rwma3I.png)
[Valorant](https://i.imgur.com/vwPwna3.png)
[Original Redesign](https://i.imgur.com/94COY0R.png)
# Download the wallpapers
[Valorant Style Download](https://atlasos.net/wallpapers/Valorant.7z)
[CS:GO Style Download](https://atlasos.net/wallpapers/CSGO.7z)
[Original Redesign Download](https://atlasos.net/wallpapers/Original_Redesign.7z)

Hopefully you like these, it tooks me some time to make but I definetely like the final product!

**Comment by hubman-star** [comment]
These links now redirect to a broken version of AtlasOS website.
